CoreFoundation/Base.subproj/CFInternal.h

```diff
@@ -216,7 +216,7 @@ extern void __CFGenericValidateType_(CFTypeRef cf, CFTypeID type, const char *fu
 #define __CFBitfield64GetValue(V, N1, N2)	(((V) & __CFBitfield64Mask(N1, N2)) >> (N2))
 #define __CFBitfield64SetValue(V, N1, N2, X)	((V) = ((V) & ~__CFBitfield64Mask(N1, N2)) | ((((uint64_t)X) << (N2)) & __CFBitfield64Mask(N1, N2)))
 
-#if __LP64__
+#if __LP64__ || defined(__ANDROID__)
```

Comment:
What is the need here? Can we document why we need to special-case Android, or can we use a compiler capability flag if that's not the case?

Reply:
This #if __LP64__ seems to be a leftover as __CFInfoType is always uint64_t in the actual definition in [CFRuntime.h](https://github.com/SwiftJava/swift-corelibs-foundation/blob/master/CoreFoundation/Base.subproj/CFRuntime.h#L199). I added this defined(__ANDROID__) to silence the many warnings telling me something wasn’t quite right. This #if should probably be removed altogether.